src/inc/quic_storage.h

```diff
@@ -35,9 +35,10 @@ void
 typedef CXPLAT_STORAGE_CHANGE_CALLBACK *CXPLAT_STORAGE_CHANGE_CALLBACK_HANDLER;
 
 typedef enum CXPLAT_STORAGE_OPEN_FLAGS {
-    CXPLAT_STORAGE_OPEN_FLAG_NONE =         0x0,
-    CXPLAT_STORAGE_OPEN_FLAG_WRITABLE =     0x1,
-    CXPLAT_STORAGE_OPEN_FLAG_DELETEABLE =   0x2
+    CXPLAT_STORAGE_OPEN_FLAG_READABLE =     0x0,
+    CXPLAT_STORAGE_OPEN_FLAG_WRITEABLE =    0x1,
+    CXPLAT_STORAGE_OPEN_FLAG_DELETEABLE =   0x2,
+    CXPLAT_STORAGE_OPEN_FLAG_CREATE =       0x4
```

Comment:
nit: Slightly strange to have all these "*able* but "create". I don't know if "creatable" is a thing, or we could go for read, write, delete, create.

Reply:
Fair point, I'll make them consistent